checkbox-provider-ce-oem/units/socketcan/jobs.pxu

```diff
@@ -0,0 +1,58 @@
+unit: template
+template-resource: com.canonical.certification::device
+template-filter: device.category == 'SOCKETCAN'
+id: ce-oem-socketcan/stress_fd_remote_{interface}
+_summary: Send stress test through CAN device {interface} (Raw, Remote, FD)
+_description:
+    Test a CAN device by sending packets using a raw socket to a remote device.
+    As a prerequisite the remote device should have can-echo-server installed so
+    as to return the predicted packet.
+category_id: com.canonical.certification::socketcan
+plugin: shell
+user: root
+estimated_duration: 10m
+flags: also-after-suspend
+imports: from com.canonical.plainbox import manifest
+requires:
+    manifest.socket_can_echo_server_running == 'True'
+    socket_can_fd == 'True'
```

Comment:
This should be `manifest.socket_can_fd`.

Reply:
thanks for the comments, fixed in latest changes.